doc/install/index.rst

```diff
@@ -9,29 +9,22 @@ MNE-Python requires Python version |min_python_version| or higher. If you've
 never worked with Python before, skip ahead to the last paragraph of this page.
 For users already familiar with Python:
 
-- If you only need MNE-Python's computational functions (not plotting), the
-  only hard dependencies are `NumPy`_ and `SciPy`_ which will be included
-  automatically when running:
+- If you only need MNE-Python's computational functions, only hard dependencies
+  will be included when running:
 
   .. code-block:: console
 
       $ pip install mne
 
-- If you plan to use MNE-Python's functions that download example datasets,
+- If you plan to use MNE-Python's functions that use HDF5-based I/O (e.g.,
+  :func:`mne.io.read_raw_eeglab`, :meth:`mne.SourceMorph.save`, etc.),
```

Comment:
Does `mne.io.read_raw_eeglab` require HDF5? This should only be necessary when MAT files are saved in the v7.3 format. In the wild, I've encountered almost exclusively files <v7.3 which don't need it.

Reply:
This is a little bit misleading -- we need `pymatreader` for that specific example, but I'm not sure this level of specificity is needed in `index.rst`. If it really bugs you I can rephrase it

Reply:
You're right, that's not necessary here.